src/SVGSkin.js

```diff
@@ -25,6 +27,12 @@ class SVGSkin extends Skin {
         /** @type {WebGLTexture} */
         this._texture = null;
 
+        /** @type {number} */
+        this._textureScale = 1;
+
+        /** @type {Number} */
+        this._maxTextureScale = 0;
```

Comment:
Since this is reset to the same value as _textureScale (i.e. 1) in `setSVG`, should this be initialized to 1 here also?